herddb-core/src/main/java/herddb/core/system/SystablesTableManager.java

```diff
@@ -37,8 +37,8 @@
     private final static Table TABLE = Table
             .builder()
             .name("systables")
-            .column("tablespace", ColumnTypes.STRING)
-            .column("table_name", ColumnTypes.STRING)
+            .column("tablespace", ColumnTypes.NOTNULL_STRING)
```

Comment:
you have to revert this change as well and the line below

Reply:
Done.